### PR TITLE
CodeIgniterMailerHandler.php

### PR DESCRIPTION
**Pull request by KashifChandio**
Mail Handler for CodeIgniter.

### CODE REVIEW

File: src/Monolog/Handler/CodeIgniterMailerHandler.php [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+ * @author Mark Garrett <mark@moderndeveloperllc.com>
+ */
+class CodeIgniterMailerHandler extends MailHandler
+{
+    /**
+     * The email addresses to which the message will be sent
+     * @var array
+     */
+    protected $_to;
+    protected $_from;
```
Comment (stof) [inline src/Monolog/Handler/CodeIgniterMailerHandler.php]: no leading underscore please

File: src/Monolog/Handler/CodeIgniterMailerHandler.php [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+    protected $encoding = 'utf-8';
+
+    /**
+     * @param string|array $to             The receiver of the mail
+     * @param string       $subject        The subject of the mail
+     * @param string       $from           The sender of the mail
+     * @param integer      $level          The minimum logging level at which this handler will be triggered
+     * @param boolean      $bubble         Whether the messages that are handled can bubble up the stack or not
+     * @param int          $maxColumnWidth The maximum column width that the message lines will have
+     */
```
Comment (stof) [inline src/Monolog/Handler/CodeIgniterMailerHandler.php]: the phpdoc should not be separated from the method

File: src/Monolog/Handler/CodeIgniterMailerHandler.php [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+    public function addHeader($headers)
+    {
+        foreach ((array) $headers as $header) {
+            if (strpos($header, "\n") !== false || strpos($header, "\r") !== false) {
+                throw new \InvalidArgumentException('Headers can not contain newline characters for security reasons');
+            }
+            $this->headers[] = $header;
+        }
+
+        return $this;
```
Comment (stof) [inline src/Monolog/Handler/CodeIgniterMailerHandler.php]: this should not be fluent IMO. It is not consistent with other parts of Monolog

**Comment by Seldaek** [comment]
Thanks for the PR but IMO this belongs in CI directly more than in monolog. Or if they don't want it then having it in a standalone package integrating the two together is an option, but having a CI-specific package in here makes little sense.

**Pull request closed without merging**